x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/hooks/use_styles.ts

```diff
@@ -45,21 +45,24 @@ export const useStyles = () => {
     }
     & .euiDataGridRowCell {
       font-size: ${euiTheme.size.m};
+
+      // Vertically center content
+      .euiDataGridRowCell__content {
+        display: flex;
+        align-items: center;
+      }
     }
-    & .euiDataGridRowCell__actions > [data-test-subj='euiDataGridCellExpandButton'] {
+    /* EUI QUESTION: Why is this being done via CSS instead of setting isExpandable: false in the columns API? */
+    & .euiDataGridRowCell__actions > .euiDataGridRowCell__expandCell {
       display: none;
     }
```

Comment:
Thanks for pointing this out! I actually wasn't aware about this option, I'll update it to use `isExpanded` in a follow up PR

Reply:
Thanks!! 🙇  I was able to take a peek at this when I was fixing the UI locally. `display: none` isn't visually breaking in this case, but it would be a very minor accessibility issue for non-keyboard screen reader users (since keyboard users can't access other cell actions except via the expansion popover).

If it's not a blocker, we'd love for this CSS to simply be removed, but we'll leave it up to you all/another PR!